### ISSUE TITLE
Account Login doesn't work

### ISSUE THREAD

**Issue by paintscape** [woothemes/woocommerce#10087]
I checked the theme, plugins and cannot find a conflict. Switched to the Storefront theme and had the same issue. I can get http://bounciesfrombohemia.com/my-account/ to work once in Windows 10 Browser and Firefox, not at all in Chrome. If you get it to work once, log out and try logging back in you will just get the blank page reloading. Once an account is created, users can not log in consistently. This is a serious bug. 

### WordPress Environment ###

Home URL: http://bounciesfrombohemia.com
Site URL: http://bounciesfrombohemia.com
WC Version: 2.4.13
Log Directory Writable: ✔  /home/paintsc1/public_html/bounciesfrombohemia/wp-content/uploads/wc-logs/
WP Version: 4.4.1
WP Multisite: –
WP Memory Limit: 64 MB
WP Debug Mode: –
Language: en_US

### Server Environment ###

Server Info: Apache
PHP Version: 5.4.28
PHP Post Max Size: 64 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
SUHOSIN Installed: –
MySQL Version: 5.5.47
Max Upload Size: 50 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 2.4.13
: 
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_termmeta: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔

### Active Plugins (10) ###

ManageWP - Worker: by ManageWP – 4.1.26
Black Studio TinyMCE Widget: by Black Studio – 2.2.8
Meta Slider - Pro Addon Pack: by Matcha Labs – 2.6.6
Meta Slider: by Matcha Labs – 3.3.6
Page Builder by SiteOrigin: by SiteOrigin – 2.2.2
UpdraftPlus - Backup/Restore: by UpdraftPlus.Com
DavidAnderson – 2.11.21.22

Visual Form Builder Pro: by Matthew Muro – 2.5.1
WooCommerce Email Inquiry & Cart Options PRO: by A3 Revolution – 1.3.3
WooCommerce: by WooThemes – 2.4.13
Yoast SEO: by Team Yoast – 3.0.7

### Settings ###

Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2

### API ###

API Enabled: ✔
API Version: 3.0.0

### WC Pages ###

Shop Base: #35 - /shop/
Cart: #36 - /cart/
Checkout: #37 - /checkout/
My Account: #38 - /my-account/

### Taxonomies ###

Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

### Theme ###

Name: Virtue - Premium
Version: 3.4.5
Author URL: http://www.kadencethemes.com/
Child Theme: ✕ – If you're modifying WooCommerce or a parent theme you didn't build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: virtue_premium/woocommerce/archive-product.php
virtue_premium/woocommerce/content-product.php
virtue_premium/woocommerce/content-product_cat.php
virtue_premium/woocommerce/content-single-product.php
virtue_premium/woocommerce/loop/loop-end.php
virtue_premium/woocommerce/loop/loop-start.php
virtue_premium/woocommerce/loop/rating.php
virtue_premium/woocommerce/loop/sale-flash.php
virtue_premium/product-searchform.php
virtue_premium/woocommerce/single-product/add-to-cart/external.php
virtue_premium/woocommerce/single-product/add-to-cart/grouped.php
virtue_premium/woocommerce/single-product/add-to-cart/simple.php
virtue_premium/woocommerce/single-product/add-to-cart/variable.php
virtue_premium/woocommerce/single-product/product-image.php
virtue_premium/woocommerce/single-product/product-thumbnails.php
virtue_premium/woocommerce/single-product/related.php
virtue_premium/woocommerce/single-product/sale-flash.php
virtue_premium/woocommerce/single-product/title.php
virtue_premium/woocommerce/single-product/up-sells.php
virtue_premium/woocommerce/single-product.php
virtue_premium/woocommerce/taxonomy-product_cat.php
virtue_premium/woocommerce/taxonomy-product_tag.php

**Comment by mikejolley** [comment]
Look at your ZenCache settings - exclude account and checkout pages. Logins themselves are handled by WP itself. As this is a caching issue (and we have no other reports/cannot reproduce) this is not a bug in WC core.

**Comment by paintscape** [comment]
I had deactivated that plugin and still had the problem. But I agree that somehow it was a caching issue. Since I needed to find a solution quickly, I went a different route by installing the ultimate member plugin and used that for customer login and woocommerce customer profile. Low and behold it works in all browsers and plays nice with the theme and woocommerce. 

The original cause is still a mystery, but the site works now.
